README.md

```diff
@@ -1,23 +1,44 @@
-# purescript-http-methods
+# Http Methods
```

Comment:
If we change the title, it'd need to be changed here as well.

Reply:
Done.